mcs/portal-ui/src/screens/Console/Console.tsx

```diff
@@ -172,8 +172,9 @@ class Console extends React.Component<
       .invoke("GET", `/api/v1/users/whoami`)
       .then(res => {})
       .catch(err => {
-        storage.removeItem("token");
-        history.push("/");
+        // Commented this as whoami API is not available. This is a temporal fix for token dissapearing issue
```

Comment:
I think we can entirely remove this until we figure out a way to validate the session